dv/uvm/icache/data/vcs/cover.cfg

```diff
@@ -0,0 +1,9 @@
++tree tb.dut
```

Comment:
Update this as needed as well, later.

Reply:
Yep, this cover.cfg file can pretty much be used verbatim for Ibex as well.